src/components/address/AddressLink.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from '@components/commons/Link'
+import classnames from 'classnames'
+
+export function AddressLink ({ address, className }: {address: string, className?: string}): JSX.Element {
+  return (
+    <div data-testid='Address.Link' className={classnames('hover:text-primary-500 cursor-pointer', className ?? '')}>
```

Comment:
A few changes for the component:

- Refactor the component into commons instead as it would be used throughout scan  
- Add testid as a props
- Following Risi's new design links should be primary-500 and underlined when hovered.

Reply:
Okay

Reply:
Also do update the addresses throughout scan (e.g. those in dftx)

Reply:
@joeldavidw I think this component could also be used for `TxIdLink` #495